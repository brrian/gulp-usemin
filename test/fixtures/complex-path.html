<!-- build:css data/css/style.css -->
<link rel="stylesheet" href="{{asset('css/clear.css')}}"/>
<link rel="stylesheet" href="{{asset('css/main.css')}}"/>
<!-- endbuild -->

<!-- build:js /data/js/app.js -->
<script src="{{asset('js/lib.js')}}"></script>
<script src="{{asset('js/main.js')}}"></script>
<!-- endbuild -->