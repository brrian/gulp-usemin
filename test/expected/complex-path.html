<link rel="stylesheet" href="{{asset('data/css/style.css')}}"/>

<script src="{{asset('/data/js/app.js')}}"></script>